ui/css/treeherder.css

```diff
@@ -638,10 +637,6 @@ th-watched-repo {
     content: ")"
 }
 
-.group-job-list {
```

Comment:
I believe we still use this class in the dom for navigation, but the styling is no longer needed.